### PR TITLE
Load paths from config file

### PR DESCRIPTION
**Pull request by Grasseh**
Also allow to pass config file either default or from args.

### CODE REVIEW

**Review by KB07 — APPROVED** [review]
Seems good to me

**Review by Lolskate — APPROVED** [review]
Clean and thorough 👍

File: demo/js/markdown/test1.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[*invoker](NodeInvoker)
+[*module](../testFacade.js)
```
Comment (Stalfy) [inline demo/js/markdown/test1.md]: This does not reflect the new syntax used in markdown files.

File: demo/js/markdown/output/output.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../src/resources/output/styles.css">
+    </head>
+    <body>
+<p><a href="NodeInvoker"><em>invoker</a>
```
Comment (Stalfy) [inline demo/js/markdown/output/output.html]: These 2 lines with be removed with @Lolskate PR, most likely to get a conflict.

**Review by Stalfy — CHANGES_REQUESTED** [review]

File: demo/js/markdown/output/output.html (hunk 2 above)
Comment (Stalfy) [reply demo/js/markdown/output/output.html]: Line 7 and 8 spicifically.

**Review by Stalfy — CHANGES_REQUESTED** [review]
Nitpick : Extract acceptance tests in their own folder.

**Review by Stalfy — APPROVED** [review]

**Pull request merged**